app/controllers/clients_controller.rb

```diff
@@ -5,9 +5,9 @@ class ClientsController < ApplicationController
 
   def index
     if params[:letter]
-      @clients = @clients.find_by_first_letter(params[:letter]).paginate(:page => params[:page])
```

Comment:
This needs to be reverted @wasi and @avinoth The controller calls `@clients.find` instead of `Client.find` because CanCanCan preloads the clients `before_filter`